t/t6030-bisect-porcelain.sh

```diff
@@ -133,21 +133,21 @@ test_expect_success 'bisect reset removes bisect state after --no-checkout' '
 
```

Comment:
I thought I read somewhere in the mail thread that this commit should not even be necessary? I *think* that the intention was to fold these changes into the respective previous commits that introduced those tests (I might very well be wrong).

OTOH it could be that this change is not even necessary any longer. I will test whether that is the case later tonight.

Reply:
> I thought I read somewhere in the mail thread that this commit should not even be necessary?

Yep: https://public-inbox.org/git/CAN0heSrWbgNEfeXj0D0SBiyC_9i++qecBcb5Kuf4WtS1wFxKDQ@mail.gmail.com/

And I also see that `GIT_TEST_GETTEXT_POISON=true ./t6030-bisect-porcelain.sh  -i` passes after reverting "t6030: make various test to pass GETTEXT_POISON tests".

So I think this patch should really be dropped (I have to admit that I do not understand how the test can pass with and without gettext "poison" when this patch is applied, but it simply seems to be unnecessary).

Reply:
Should I drop this commit [Patch 8/8]?

Reply:
Yes, please 🙂